public-ui/src/app/views/ehr-view/ehr-view.component.html

```diff
@@ -237,7 +237,7 @@ <h6 *ngIf="searchText.value.length > 0" class="primary-display medline-link">Int
                     </div>
                   </div>
                   <ng-container *ngIf="graphToShow === graphType.Sources; else NonSourceCharts">
-                    <div class="row" *ngIf="r.vocabularyId === 'SNOMED'; else NonRolledUp">
+                    <div class="row" *ngIf="r.vocabularyId === 'SNOMED' && (r.domainId === 'Condition' || r.domainId === 'Procedure'); else NonRolledUp">
```

Comment:
Only condition and procedure snomed conditions are rolled up, so display tree only for those concepts.